### PR TITLE
fix: Google Spam Polciy alert breaking embed layout in popup mode

### LINKED ISSUE #12151: [CAL-2663] Embed: Google New Spam Policy alert breaking embed layout
https://discord.com/channels/1125390093386010654/1167188597586264184

image

<sub>[CAL-2663](https://linear.app/calcom/issue/CAL-2663/embed-google-new-spam-policy-alert-breaking-embed-layout)</sub>

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Fixes #12151

Loom: https://www.loom.com/share/fbb38d07327142c8ba573feed497ff13

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

See loom

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by hariombalhara** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11774** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/11774" target="_blank">Graphite</a> 
    * **PR #12153** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/12153" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/calcom/cal.com/12153?utm_source=stack-comment).

File: apps/web/pages/booking/[uid].tsx [hunk 1]
```diff
@@ -342,14 +342,17 @@ export default function Success(props: SuccessProps) {
           <div
             className={classNames(
               shouldAlignCentrally ? "text-center" : "",
-              "flex items-end justify-center px-4 pb-20 pt-4  sm:block sm:p-0"
+              "flex items-end justify-center px-4 pb-20 pt-4 sm:flex sm:p-0"
```
Comment (hariombalhara) [inline apps/web/pages/booking/[uid].tsx:345]: I don't know why we were going from flex to block in sm

File: apps/web/pages/booking/[uid].tsx [hunk 2]
```diff
@@ -342,14 +342,17 @@ export default function Success(props: SuccessProps) {
...
             )}>
             <div
-              className={classNames("my-4 transition-opacity sm:my-0", isEmbed ? "" : " inset-0")}
+              className={classNames(
+                "main my-4 flex flex-col transition-opacity sm:my-0 ",
```
Comment (hariombalhara) [inline apps/web/pages/booking/[uid].tsx:349]: This one is also flex now

Comment (hariombalhara) [inline apps/web/pages/booking/[uid].tsx:349]: Switched main to be this wrapper element that contains alert as well.

**Review by sean-brydon — APPROVED** [review]
LGTM - as discussed in PMs. Lets tackle the scrolling improvements in a follow up

**Pull request merged**